server/api/db/sqldb/db.py

```diff
@@ -1753,10 +1754,18 @@ def store_function(
             function.setdefault("metadata", {})["name"] = name
         fn = self._get_class_instance_by_uid(session, Function, name, project, uid)
         if not fn:
+            foo_spec = function.get("spec", {}).get("foo")
+            func_foo_spec = None
+            if foo_spec:
+                func_foo_spec = FunctionSpecFoo(
+                    data=foo_spec,
+                )
```

Comment:
```suggestion
            func_foo_spec = FunctionSpecFoo(data=foo_spec) if foo_spec else None
```

Reply:
I'm not a biq fan of one line expressions like this